app/assets/stylesheets/all/questions.css.scss

```diff
@@ -102,3 +102,70 @@ ul.tags {
   max-height: 93px;
   overflow: auto;
 }
+
+
+// TODO: Ensure styles apply without increased specificity
+#questions-header.questions-header {
+
+  span {
+    margin-right: 10px;
+  }
+
+  a {
+    i {
+    color: inherit;
+    }
+  }
+}
+.questions-list {
+  padding-left: 0;
+
+  li {
+    list-style-position: inside;
+  }
+
+  .question {
```

Comment:
For this stuff see the styles in option_sets.css.scss. The stuff for draggable list is pretty clearly marked. Perhaps you can split it out into its own file and then reuse?

With Rails, if you make a new .css.scss file and put it in the same dir, it will get auto-loaded.